components.d.ts

```diff
@@ -16,9 +16,6 @@ declare module '@vue/runtime-core' {
     NForm: typeof import('naive-ui')['NForm']
     NFormItem: typeof import('naive-ui')['NFormItem']
     NIcon: typeof import('naive-ui')['NIcon']
-    NInput: typeof import('naive-ui')['NInput']
-    NInputGroup: typeof import('naive-ui')['NInputGroup']
-    NInputGroupLabel: typeof import('naive-ui')['NInputGroupLabel']
```

Comment:
Why did you modify the d.ts files for naive-ui? Has is anything to do with tracking?

Reply:
I didn't modify it: this is generated by the [auto-import](https://github.com/antfu/unplugin-auto-import).

Reply:
It's got nothing to do with the current issue